cloud/deploys/dev_azure/civiform_config.example.sh

```diff
@@ -1,36 +1,52 @@
-# More details and documentation for these are consolidated in the deploy repo,
+# More details and documentation for these are consolidated in the deploy repo:
 # https://github.com/civiform/civiform-deploy/blob/main/civiform_config.example.sh
+# Instructions for local dev deploys are available here:
+# https://docs.civiform.us/contributor-guide/developer-guide/dev-azure
 export CIVIFORM_CLOUD_PROVIDER="azure"
 export CIVIFORM_MODE="dev"
 export CIVIC_ENTITY_SHORT_NAME="Sooschester"
 export CIVIC_ENTITY_FULL_NAME="City of Sooschester"
-export CIVIC_ENTITY_SUPPORT_EMAIL_ADDRESS="CHANGE ME"
+export CIVIC_ENTITY_SUPPORT_EMAIL_ADDRESS="civiform-azure-staging-email@googlegroups.com"
```

Comment:
I think we had left these as CHANE ME to prevent spam but that doesn't seem to be a big problem atm

Reply:
Acknowledged